Tests/UnboxTests.swift

```diff
@@ -1358,6 +1358,27 @@ class UnboxTests: XCTestCase {
         }
     }
     
+    func testUnboxingArrayOfStringTransformedToInt() {
+        let dictionary: UnboxableDictionary = ["intArray": ["123", "456", "789"]]
+        
+        struct ModelA: Unboxable {
+            let intArray: [Int]
+            init(unboxer: Unboxer) {
+                intArray = unboxer.unbox("intArray")
```

Comment:
I was once as well and wouldn't mind being one again ;) I'm also including a test for a bad array ["123", "abc", "456"]